src/graphql/types/userTypes.ts

```diff
@@ -3,12 +3,22 @@ import { QueryOptions } from "./queryTypes";
 export type SignUpData = {
   email: string;
   password: string;
+  civility: string;
   firstName: string;
   lastName: string;
   organizationName: string;
   functionTitle: string;
+  address: string;
   city: string;
+  activity: string;
   country: string;
+  displayEmail: string;
+}
+
+export enum Civility {
+  MS = "Ms.",
+  MR = "Mr.",
+  DR = "Dr.",
```

Comment:
"Other" option ?

Reply:
Other is not present in the options, I did it like this to match what was expected